### ISSUE TITLE
also implement h265 and AV1

### ISSUE THREAD

**Issue by Benman2785** [ruffle-rs/ruffle#15403]
Hey,

when implementing flv with h264 it would be awesome to include h265 and AV1 - as both are supported by RTMP enhanced
and not only mp3 & aac but also AC3 would be nice

**Comment by torokati44** [comment]
> it would be awesome to include h265 and AV1 - as both are supported by RTMP enhanced

Yes, but AFAIK not by Flash Player, therefore, they are out of scope. Not to mention, H265 too is riddled by patents, and I'm not aware of an "OpenH265" type of royalty deal.

> but also AC3 would be nice

Similarly, I'm not aware of Flash Player supporting this format.

**Comment by Croworbit** [comment]
should this be closed?

**Comment by Benman2785** [comment]
why not extend over FlashPlayer. Especially AV1 would be a great addition for streamers using flash

**Comment by torokati44** [comment]
Because there is no existing content that makes use of it, nor any reason or easy way to make new content that would.
